tests/pytorch/r2.1/llama2-model.libsonnet

```diff
@@ -160,7 +218,8 @@ local utils = import 'templates/utils.libsonnet';
   },
 
   configs: [
-    llama2_inference + v4_8 + common.Functional + timeouts.Hours(3) + pjrt,
-    llama2_training + v4_8 + common.Functional + timeouts.Hours(3) + hf,
+    llama2_inference + v4_8 + common.Functional + timeouts.Hours(3) + infer,
+    // llama2_training + v4_8 + common.Functional + timeouts.Hours(3) + fsdp,
```

Comment:
Do we not want `fsdp`? If not, we can just remove it.

Reply:
thanks for pointing it, we don't want fsdp here for now, will delete

Reply:
For confirm, do we want to delete config command only and keep setup of fsdp, or we want to delete everything about fsdp

if we keep setup of fsdp here, we could bring it back easily in the future

Reply:
Let's just remove it, we can always just refer to this PR if we want to bring it back.

Reply:
Thanks, updated